aria/orchestrator/workflows/api/task.py

```diff
@@ -140,13 +140,18 @@ def __init__(self,
         self.arguments = modeling_utils.merge_parameter_values(arguments,
                                                                operation.arguments,
                                                                model_cls=models.Argument)
-        if getattr(self.actor, 'outbound_relationships', None) is not None:
```

Comment:
Why not import models? This should have been explained in comments. Duck typing seems very inaccurate and prone to errors.